Release 3.3.40 and 4.6.10 (#2467)
ibexa/documentation-developer: docs/update_and_migration/from_3.3/update_from_3.3.md

    ``` sql
    psql <database_name> < vendor/ibexa/installer/upgrade/db/postgresql/ibexa-3.3.33-to-3.3.34.sql
    ```

### v3.3.40

> A command to deal with the duplicated entries caused by [IBX-8562](https://issues.ibexa.co/browse/IBX-8562) will be available soon.

Reviewer: ```suggestion
A command to deal with duplicated entries, as reported in [IBX-8562](https://issues.ibexa.co/browse/IBX-8562), will be available soon.
```

> A command dealing with the duplicated entries caused by [IBX-8562](https://issues.ibexa.co/browse/IBX-8562) will be provided here in the near future.

Reviewer 2: A command to deal with the duplicated entries caused by [IBX-8562](https://issues.ibexa.co/browse/IBX-8562) will be available soon.

> A command dealing with the duplicated entries caused by [IBX-8562](https://issues.ibexa.co/browse/IBX-8562) will be provided here in the near future.

Reviewer 2: ```suggestion
A command to deal with the duplicated entries caused by [IBX-8562](https://issues.ibexa.co/browse/IBX-8562) will be available soon.
```

> A command dealing with the duplicated entries caused by [IBX-8562](https://issues.ibexa.co/browse/IBX-8562) will be provided here in the near future.

Reviewer: ```suggestion
A command to deal with duplicated entries, as reported in [IBX-8562](https://issues.ibexa.co/browse/IBX-8562), will be available soon.
```
